### PR TITLE
Changes to build badge to reflect changes in azure-pipelines due to

### PR DESCRIPTION
**Pull request by nedrebo**
repository name change

### CODE REVIEW

**Comment by nedrebo** [comment]
Bump @eskaur @knatten

**Comment by eskaur** [comment]
> to reflect changes in azure-pipelines due to #10
But this is PR #10 (?)

**Comment by eskaur** [comment]
> > to reflect changes in azure-pipelines due to #10
> > But this is PR #10 (?)

Haha, reading that headline a lil' too quick

File: README.md [hunk 1]
```diff
@@ -27,5 +27,5 @@ The list of packages to be uploaded is found in `scripts/generate_all_pkgbuilds.
 
 Extend those lists when Zivid releases more packages.
 
-[ci-badge]: https://img.shields.io/azure-devops/build/zivid-devops/376f5fda-ba80-4d6c-aaaa-cbcd5e0ad6c0/2/master.svg
-[ci-url]: https://dev.azure.com/zivid-devops/arch-linux-pkgbuild-generator/_build/latest?definitionId=2&branchName=master
\ No newline at end of file
+[ci-badge]: https://img.shields.io/azure-devops/build/zivid-devops/83d24458-0938-4d05-a0d0-104cd8dff5e3/7
+[ci-url]: https://dev.azure.com/zivid-devops/zivid-arch-linux-pkgbuild-generator/_build/latest?definitionId=7&branchName=master
```
Comment (eskaur) [inline README.md]: Out of curiosity, why is it changed from 2 to 7?

File: README.md [hunk 2]
```diff
@@ -27,5 +27,5 @@ The list of packages to be uploaded is found in `scripts/generate_all_pkgbuilds.
 
 Extend those lists when Zivid releases more packages.
 
-[ci-badge]: https://img.shields.io/azure-devops/build/zivid-devops/376f5fda-ba80-4d6c-aaaa-cbcd5e0ad6c0/2/master.svg
-[ci-url]: https://dev.azure.com/zivid-devops/arch-linux-pkgbuild-generator/_build/latest?definitionId=2&branchName=master
\ No newline at end of file
+[ci-badge]: https://img.shields.io/azure-devops/build/zivid-devops/83d24458-0938-4d05-a0d0-104cd8dff5e3/7
```
Comment (eskaur) [inline README.md]: Where does this hash come from?

Comment (nedrebo) [reply README.md]: Click on Azure here for doc on how to find it: https://shields.io/category/build

File: README.md (hunk 1 above)
Comment (nedrebo) [reply README.md]: Click on Azure here for doc on how to find this number: https://shields.io/category/build

**Comment by eskaur** [comment]
Btw, you need to go into the repo settings (branch protections) and change the "Required" checks. It seems to want one of the old tests.

**Comment by nedrebo** [comment]
> Btw, you need to go into the repo settings (branch protections) and change the "Required" checks. It seems to want one of the old tests.

Done

**Pull request merged**